app/sensors/DropdownList.js

```diff
@@ -306,25 +298,27 @@ export default class DropdownList extends Component {
 			if(value) {
 				result = [];
 				value.map((item) => {
-					result.push(item.value);
+					if (_.isArray(item.value)) {
```

Comment:
Instead of using lodash, you can use native ES6 method: `Array.isArray()`. But in this case, you can simply check for the boolean prop `multipleSelect`. It will have better performance and for type checking, we already have `defaultSelected: React.PropTypes.array` present in `MultiDropdownList`'s propTypes.